src/vscode-dts/vscode.d.ts

```diff
@@ -19379,9 +19400,10 @@ declare module 'vscode' {
 		role: LanguageModelChatMessageRole;
 
 		/**
-		 * The content of this message.
+		 * A string or heterogeneous array of things that a message can contain as content. Some parts may be message-type
+		 * specific for some models.
 		 */
-		content: string;
+		content: string | (string | LanguageModelToolResultPart | LanguageModelToolCallPart)[];
```

Comment:
Was this the intent to merge content2 and content?

Reply:
yes. it's one of the things where TS didn't let me redefine `content`